pkg/kubectl/describe.go

```diff
@@ -1201,7 +1190,55 @@ func (d *HorizontalPodAutoscalerDescriber) Describe(namespace, name string) (str
 	})
 }
 
-func filterNonRunningPods(pods []*api.Pod) []*api.Pod {
+func describeNodeResource(pods []*api.Pod, node *api.Node, out io.Writer) {
+	nonTerminatedPods := filterTerminatedPods(pods)
+	fmt.Fprintf(out, "Non-terminated Pods:\t(%d in total)\n", len(nonTerminatedPods))
+	fmt.Fprint(out, "  Namespace\tName\t\tCPU Requests\tCPU Limits\tMemory Requests\tMemory Limits\n")
+	fmt.Fprint(out, "  ─────────\t────\t\t────────────\t──────────\t───────────────\t─────────────\n")
+	totalMilliCPUReq := int64(0)
+	totalMemoryReq := int64(0)
+	fractionPodCPUReq := float64(0)
+	fractionPodMemoryReq := float64(0)
+	fractionTotalCPUReq := float64(0)
+	fractionTotalMemoryReq := float64(0)
+	totalMilliCPULimit := int64(0)
+	totalMemoryLimit := int64(0)
+	fractionPodCPULimit := float64(0)
+	fractionPodMemoryLimit := float64(0)
+	for _, pod := range pods {
+		podTotalMilliCPUReq := int64(0)
+		podTotalMemoryReq := int64(0)
+		podTotalMilliCPULimit := int64(0)
+		podTotalMemoryLimit := int64(0)
+
+		for ix := range pod.Spec.Containers {
+			requests := pod.Spec.Containers[ix].Resources.Requests
+			podTotalMilliCPUReq += requests.Cpu().MilliValue()
+			podTotalMemoryReq += requests.Memory().Value()
+
+			limits := pod.Spec.Containers[ix].Resources.Limits
+			podTotalMilliCPULimit += limits.Cpu().MilliValue()
+			podTotalMemoryLimit += limits.Memory().Value()
+		}
+		totalMilliCPUReq += podTotalMilliCPUReq
+		totalMemoryReq += podTotalMemoryReq
+		totalMilliCPULimit += podTotalMilliCPULimit
+		totalMemoryLimit += podTotalMemoryLimit
+		fractionPodCPUReq = float64(podTotalMilliCPUReq) / float64(node.Status.Capacity.Cpu().MilliValue()) * 100
+		fractionPodMemoryReq = float64(podTotalMemoryReq) / float64(node.Status.Capacity.Memory().Value()) * 100
+		fractionPodCPULimit = float64(podTotalMilliCPULimit) / float64(node.Status.Capacity.Cpu().MilliValue()) * 100
+		fractionPodMemoryLimit = float64(podTotalMemoryLimit) / float64(node.Status.Capacity.Memory().Value()) * 100
+		fmt.Fprintf(out, "  %s\t%s\t\t%dm (%d%%)\t%dKi (%d%%)\t%dm (%d%%)\t%dKi (%d%%)\n", pod.Namespace, pod.Name,
+			podTotalMilliCPUReq, int64(fractionPodCPUReq), podTotalMilliCPULimit, int64(fractionPodCPULimit), podTotalMemoryReq/1000, int64(fractionPodMemoryReq), podTotalMemoryLimit/1000, int64(fractionPodMemoryLimit))
+	}
+	fmt.Fprint(out, "Allocated resources:\n  CPU Requests\tCPU Limits\tMemory Requests\tMemory Limits\n")
+	fmt.Fprint(out, "  ────────────\t──────────\t───────────────\t─────────────\n")
+	fractionTotalCPUReq = float64(totalMilliCPUReq) / float64(node.Status.Capacity.Cpu().MilliValue()) * 100
+	fractionTotalMemoryReq = float64(totalMemoryReq) / float64(node.Status.Capacity.Memory().Value()) * 100
+	fmt.Fprintf(out, "  %dm (%d%%)\t%dm\t%dKi (%d%%)\t%dKi\n", totalMilliCPUReq, int64(fractionTotalCPUReq), totalMilliCPULimit, totalMemoryReq/1000, int64(fractionTotalMemoryReq), totalMemoryLimit/1000)
```

Comment:
I also would prefer you lean on the resource.Quantity object to handle serialization of the resource values rather than you yourself doing the 'm' or 'Ki' suffix piece.  We should be keeping all printer decisions for resources local to the resource.Quantity object.